### PR TITLE
sanity fix missing elements under list type attribute

### PR DESCRIPTION
**Pull request by KB-perByte**
##### SUMMARY

sanity fix missing elements under list type attribute

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by rohitthakur2590 — APPROVED** [review]

**Comment by goneri** [comment]
Can you add a `Depends-On: https://github.com/ansible/network-ee/pull/151` and do a recheck?

**Comment by komaldesai13** [comment]
recheck

**Comment by goneri** [comment]
> Depends-On: [ansible/network-ee#151](https://github.com/ansible/network-ee/pull/151)

This has to be done in the PR description.

**Comment by goneri** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

File: changelogs/fragments/fix_sanity_trivial.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+trivial:
```
Comment (KB-perByte) [inline changelogs/fragments/fix_sanity_trivial.yml:2]:
```suggestion
minor_change:
```

**Comment by komaldesai13** [comment]
recheck

**Comment by KB-perByte** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by komaldesai13** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Pull request merged**